### ISSUE TITLE
Simplify skeleton a little bit

### ISSUE THREAD

**Issue by dwertheimer** [NotePlan/plugins#254]
*** Super Low Priority ***

I just created a new plugin using the CLI. It definitely works (great). But I think we can make a couple of minor tweaks to simplify things for people just starting out:
1) I would suggest getting rid of the "support" folder and file within. Especially because the file is called "hello-world.js" it's unnecessarily confusing. 
![Screen Shot 2022-03-28 at 9 48 31 AM](https://user-images.githubusercontent.com/8949588/160448253-bd40fe51-05cc-4c8d-ab50-c3bd5bd72e68.jpg)
If you want to keep it, maybe rename that file "helperFunctions.js" or something like that **and** there should be tests for that file if it's there. But for my money, I'd get rid of it
2) imports with the shorthand `@` need to be fixed/replaced with the long-hand
3) the file helloWorld.js should actually be called "NPhelloWorld.js" to signify it has NP-specific code in it.

**Comment by dwertheimer** [comment]
I started a new plugin. Seeing silent fails in my code with no console logging.
I suggest we have the plugin entrypoints start with try/catch statements. 
@mikeerickson can you pls put it in the sample skeleton?
@EduardMe can you pls update the plugin docs to show this in even the most basic plugin?
Here's your sample code, modified to have the try/catch
`
function moveNote() {
  try {
    console.log("Hello World")    
  } catch (error) {
    console.log("Plugin code error: \n"+JSON.stringify(error))
  }
}
`

**Comment by jgclark** [comment]
`console.log()`? Could we not also move to the log()/logWarn()/logError() calls here?

**Comment by dwertheimer** [comment]
Yes! and let's also add another default function in index.js called "init". It can be empty with a comment for now, but would be good for completeness.

**Comment by dwertheimer** [comment]
Also, maybe we import clo by default as well

**Comment by dwertheimer** [comment]
Created a plugin today and was reminded of these changes I wanted to make.  @mikeerickson. Pushing up a PR.
